src/widgets/setupdialog.cpp

```diff
@@ -381,8 +381,8 @@ void SetupDialog::setTarsnapVersion(QString versionString)
     }
     else
     {
-        _ui->clientVersionLabel->setText(tr("Tarsnap CLI version ") +
-                                         _tarsnapVersion + tr(" detected."));
+        _ui->clientVersionLabel->setText("Tarsnap CLI version " +
```

Comment:
Make sure you keep text encapsulated in tr() to keep text translatable. I like the checkmark.